Mirror pine and wooden doors

Charyline opened the pull request:
Pine doors, wooden doors, light wooden doors and dark wooden doors can be reversed using a mallet.
Reversed versions can be dismantled using an adze.

Charyline committed:
Added open and closed left pine and wooden doors, and transitions for open and close, mallet switch and adze dismantlement

Charyline committed:
fixed rope mirror on left pine door

TanyaPegasus commented:
You've done a great job with this. That was a good catch with the flip on the rope sprites too.

I have a few notes. Really it's only a few things. It just may seem longer because of the explanations.

- `+blocksMoving` is a tag used to control whether something blocks animals from moving, even if it doesn't block people. It's used on the springy doors because they're considered more like gates in that sense (you don't want animals escaping every time someone just walks near the door). In this case, because these are regular doors, you should remove that tag.

- I suspect there was some confusion about that tag, which is why the next part was missed. The wooden doors aren't currently blocking movement for people. 

- The next note is something already present in the current mirrored doors. The shadow sprites are also being mirrored. It looks a bit odd, and causes strange overlap when walls are placed next to the doors. The pine door is great, but could you adjust the shadows for the closed wooden doors so they face the same way as the other version? It would be great if you could do that for any mirrored doors already in the game as well.

Charyline committed:
changed mirrored shadow under left oriented doors to non mirrored for wooden, light wooden, dark wooden, and springy wooden, springy dark wooden, springy light wooden doors

Charyline committed:
tag existing and new flipped doors to flipped instead of left, add a springy and non springy flipped wooden door with plain wreath

Charyline committed:
add plain, rose, berry and bell wreath flipped doors for both springy and non springy, normal, light and dark wooden doors

Charyline committed:
Fix some pixel differences on wooden door position and rose wreath position

risvh committed:
More one-pixel changes

Essentially, just pick one version of position, and make sure all the wreaths match it.

risvh commented:
I did the one-pixel changes. There are two more issues:
- Now that there are flipped versions of these doors, can you make the installed doors "no-flip"? (If it is NOT no-flip, e.g. a sharp stone, when you drop it on a tile from the right side vs left side, the stone can look flipped.) A normal door flipped, vs a flipped door, has confused me.
- Can you please also add the transitions to put wreaths on flipped doors and flipped spring doors? Thanks

Charyline committed:
Add transitions to add wreath on all wooden flipped doors and remove wreath on all flipped doors using shears. Fix double shadow on Dark flipped door with plain wreath

risvh committed:
Adjusted some wreath sounds
